libs/core/src/lib/popover/popover.component.ts

```diff
@@ -370,10 +376,12 @@ export class PopoverComponent extends BasePopoverClass
 
     /** @hidden */
     private _applyWidthOverlay(): void {
+        const maxWidthLimit = this.maxWidth ? this.maxWidth : MAX_BODY_SIZE;
```

Comment:
Do we need separately set the width to overlay and on `.fd-popover__popper` level?

Reply:
Right now styles are applied only on `.fd-popover__popper` level